### PR TITLE
fix: update @smoya/asyncapi-adoption-metrics to 2.4.7 version

### PR DESCRIPTION
**Pull request by peter-rr**
This PR will update "asyncapi-adoption-metrics" dependency to latest version.

cc @KhudaDad414 @Amzani

### CODE REVIEW

**Review by KhudaDad414 — APPROVED** [review]
LGTM!

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🎸

File: package.json
Comment (Shurtu-gal) [inline package.json:1]: Should we update to the latest [`2.4.7`](https://github.com/smoya/asyncapi-adoption-metrics/releases/tag/v2.4.7) version ?

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

File: package.json
Comment (peter-rr) [reply package.json:1]: Sure! Gonna update it 👍

**Review by Shurtu-gal — APPROVED** [review]

**Pull request merged**